raiden-dapp/src/components/dialogs/UdcDepositDialog.vue

```diff
@@ -135,7 +135,17 @@ export default class UdcDepositDialog extends Vue {
   }
 
   get valid(): boolean {
-    return /^[1-9]\d*$/.test(this.defaultUtilityTokenAmount);
+    const utilityTokenAmount = Number(this.defaultUtilityTokenAmount);
+    const utilityTokenBalance = Number(this.utilityTokenBalance);
+    const amountRegEx = /^\d+\.?\d*$/;
```

Comment:
You don't need regex if you're using the utility, it'll throw if it fails. You may need to handle though.

Reply:
I think I need the RegEx for the testnet since a user can enter any value they want?

Reply:
Let them, `parseUnits` should throw if it's invalid, you then just handle it.